### ISSUE TITLE
Where can we get the CSV files used in the script?

### ISSUE THREAD

**Issue by missaugustina** [eapower/BuildingBigness#1]
I really enjoyed this paper and am seeking to apply some of the methods you've used for your quantitative analysis. I was looking at your script and noticed the mention of a few different CSV files. Are they available for download anywhere?

**Comment by eapower** [comment]
Hi there. Glad you enjoyed the paper! The CSV files aren't publicly available. I have a data usage agreement that I ask people to sign before sending the files directly -- this is because I want to make sure there aren't any attempts at de-anonymizing the data. If you're interested in the methods just to see how they're employed, an alternative to that would be just to look at other tutorials for ERGMs. For that, I can recommend this tutorial: https://eehh-stanford.github.io/SNA-workshop/ergm-intro.html

And along with it, a tutorial that Elspeth Ready and I wrote up about GWESP and other GW* terms:  https://eehh-stanford.github.io/SNA-workshop/ergm-predictions.html

If you're interested in getting the data, though, just let me know, and I'll send the data agreement form for signing!

**Comment by missaugustina** [comment]
Thanks! I will send an email to the address in the README. I've forked your repo and am refactoring this into an Rmd document for my own understanding. I'm familiar with ERGM's but haven't actually used them beyond tutorials I've already done. I completed the Datacamp iGraph tutorial and it was really helpful. I will check out the one you created as well! I'm currently studying social capital in open source communities and am exploring how different Github events correlate to behavior (current body of research simply assumes they correlate to the expected or documented behavior). I am thinking about how to apply your mixed methods approach to the open source communities I'm currently studying. A first step for me is to be able to reproduce your results so I can better understand how the qualitative pieces tie into the quantitative at an implementation level.

**Comment by eapower** [comment]
Sounds good! As a brief note on methods, if you have temporal data, then other tools may be more appropriate, e.g. [temporal ERGMs](https://statnet.github.io/Workshops/tergm_tutorial.html) or SAOMs (stochastic actor-oriented models, also often known as Siena models), which can be done in R with [rSiena](http://www.stats.ox.ac.uk/~snijders/siena/)).

**Comment by missaugustina** [comment]
Just following up, I'm sandwiching working on this in between other things! I created a fork and am experimenting with moving chunks of your R script into an Rmd notebook so I can better understand what each section is doing. The tutorials you linked on ERGM have been super helpful. Honestly this paper is my favorite example of mixed methods I've seen this year and I love the underlying arch of the story as well!! 

I'm still figuring out where I'm going with this and how it connects to your work, but this is what I'm working on - https://github.com/IBM/visualizing-github-networks/blob/wip/visualizing-github-networks.Rmd  My plan is to use R-ladies since it's a small community of friendly folks to identify values similar to your own methodology. What I'm curious about is if the digital residue/material culture is highly correlated to network centrality of "generosity" for example. So in this case, are there patterns of Github events that could indicate a high-value social capital exchange? I am still very much in the idea phase, poking around with code at this point and doing ethnography prep work. For the particular thing I linked I might just do something overly simple just to get a concrete milestone out the door, and then continue to release additional versions as I make progress in my experiments. 

Anyways, thanks so much! I'll let you know if I have questions or run into any issues.
